.gitignore

```diff
@@ -25,6 +25,6 @@ tests/e2e/config/local-*
 
 /vendor/
 
-dist
+build
```

Comment:
Is `dist` still needed for "legacy" js file builds?

Reply:
From what I can see, this extension doesn't have legacy JS files, so `dist` never exists.